Formula/n/nwchem.rb

```diff
@@ -74,8 +88,15 @@ def install
       ENV["LIBXC_INCLUDE"] = Formula["libxc"].opt_include.to_s
       os = OS.mac? ? "MACX64" : "LINUX64"
       system "make", "nwchem_config", "NWCHEM_MODULES=all python gwmol", "USE_MPI=Y"
+      cd "tools" do
+        system "make", "NWCHEM_TARGET=#{os}", "USE_MPI=Y"
+      end
+      mkdir_p "../bin/#{os}"
+      system ENV.cc, "config/depend.c", "-o", "../bin/#{os}/depend.x"
```

Comment:
Why are these new steps needed and not part of the makefile? It was building without those, and they seem very fragile.

Reply:
> Why are these new steps needed and not part of the makefile? It was building without those, and they seem very fragile.

I agree that this is a bit of a hack. However, this two step process make the compilations much faster (we just managed to get the build time for macos x86_64 under 90 minutes).  More makefile changes are in the works.

Reply:
@fxcoudert For reference: @edoapra is a maintainer for this software. They often open PR's to improve the build and experience for users of this software. I'm not terribly concerned about the fragility here, because I'm confident they actually are working on Makefile changes given my past experience with their PR's.

Reply:
Fine with me then 👍🏻. Would be good to have a Ruby comment noting that upstream plans on removing the need for this in a future version?